<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupNameConflictDialog</class>
 <widget class="QDialog" name="BackupNameConflictDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>319</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>0</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background-color: #FAFAFA;
}

QTextBrowser
{
font-family: 'SF Pro';
font-style: normal;
font-weight: 400;
font-size: 14px;
line-height: 20px;
background: transparent;
}
QLabel
{
font-family: 'SF Pro';
font-style: normal;
font-weight: 400;
font-size: 14px;
line-height: 20px;
background: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <property name="leftMargin">
      <number>24</number>
     </property>
     <property name="rightMargin">
      <number>44</number>
     </property>
     <property name="bottomMargin">
      <number>16</number>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="lTitle">
       <property name="styleSheet">
        <string notr="true">*{
font-family: 'Inter';
font-style: normal;
font-weight: 500;
font-size: 16px;
line-height: 24px;
}</string>
       </property>
       <property name="text">
        <string>Rename folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lConflictText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">*
{
font-family: 'Inter';
}</string>
       </property>
       <property name="text">
        <string notr="true">Conflict text</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lToBackupCenterText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">*
{
color: #AA1A00;
font-family: 'Inter';
}</string>
       </property>
       <property name="text">
        <string notr="true">to backup center text</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2" alignment="Qt::AlignTop">
      <widget class="QLabel" name="lWarningIcon">
       <property name="pixmap">
        <pixmap resource="../../../../gui/Resources_macx.qrc">:/images/icon_warning.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">#wConflictZone
{
background-color: #ECECEC;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="wConflictZone">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>560</width>
        <height>84</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="scrollAreaLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>24</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>24</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignLeft">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../gui/Resources_macx.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BackupNameConflictDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BackupNameConflictDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
